tests/test_cli.py

```diff
@@ -181,43 +180,43 @@ def test_search_0(self):
         self.assertIsInstance(res, dict)
         self.assertIsInstance(res['conda'], list)
         self.assertIsInstance(res['conda'][0], dict)
-        keys = ('build', 'channel', 'extracted', 'features', 'fn',
-                'installed', 'version')
+        keys = ('build', 'channel', 'fn', 'version')
         for key in keys:
             self.assertIn(key, res['conda'][0])
 
         stdout, stderr, rc = run_inprocess_conda_command('conda search * --json')
-        assert json.loads(stdout.strip())['exception_name'] == 'CommandArgumentError'
+        # assert json.loads(stdout.strip())['exception_name'] == 'CommandArgumentError'
+        # assert len(json.loads(stdout.strip())['anaconda']) >= 1
         assert stderr == ''
-        assert rc > 0
+        assert rc == None
 
-        res = capture_json_with_argv('conda search --canonical --json')
-        self.assertIsInstance(res, list)
-        self.assertIsInstance(res[0], text_type)
+        # res = capture_json_with_argv('conda search --canonical --json')
+        # # self.assertIsInstance(res, list)
+        # self.assertIsInstance(res[0], text_type)
 
     @pytest.mark.integration
     def test_search_1(self):
         self.assertIsInstance(capture_json_with_argv('conda search ipython --json'), dict)
 
-    @pytest.mark.integration
-    def test_search_2(self):
-        self.assertIsInstance(capture_json_with_argv('conda search --unknown --json'), dict)
-
+    # @pytest.mark.integration
+    # def test_search_2(self):
+    #     self.assertIsInstance(capture_json_with_argv('conda search --unknown --json'), dict)
+    #
     @pytest.mark.integration
     def test_search_3(self):
         self.assertIsInstance(capture_json_with_argv('conda search --json --use-index-cache'), dict)
-
-    @pytest.mark.integration
-    def test_search_4(self):
-        self.assertIsInstance(capture_json_with_argv('conda search --json --outdated'), dict)
-
-    @pytest.mark.integration
-    def test_search_5(self):
-        self.assertIsInstance(capture_json_with_argv('conda search -c https://conda.anaconda.org/conda --json nose'), dict)
-
-    @pytest.mark.integration
-    def test_search_6(self):
-        self.assertIsInstance(capture_json_with_argv('conda search -c https://conda.anaconda.org/conda --override-channel --json nose'), dict)
+    #
+    # @pytest.mark.integration
+    # def test_search_4(self):
+    #     self.assertIsInstance(capture_json_with_argv('conda search --json --outdated'), dict)
+    #
+    # @pytest.mark.integration
+    # def test_search_5(self):
+    #     self.assertIsInstance(capture_json_with_argv('conda search -c https://conda.anaconda.org/conda --json nose'), dict)
+    #
+    # @pytest.mark.integration
+    # def test_search_6(self):
+    #     self.assertIsInstance(capture_json_with_argv('conda search -c https://conda.anaconda.org/conda --override-channel --json nose'), dict)
 
```

Comment:
Just delete all the tests you commented out here.  But also, add at least two new tests.

    conda search nose
    conda search nose --info